### DISCUSSION TITLE
Render remote page in a note?

### DISCUSSION THREAD

**Discussion by gerroon** [TriliumNext/Trilium#2162] [Ideas]
Hi

Is it possible to embed a remote page in a note so that I can get realtime updates from that page? I want to embed some project documentations from my project planning server.

**Comment by zadam** [comment]
You can just use iframe in the rendered page:

```html
<iframe src="https://www.remotepage.com/"></iframe>
```

**Reply by gerroon** [reply]
Thanks that sounds great but the documentation about it is a bit confusing. 

>renderNote - notes of type "render HTML note" will be rendered using a code note (HTML or script) and it is necessary to point using this relation to which note should be rendered

How does this work exactly? What is pointing in this respect? Where do I enter the iframe snippet?

![trilium_0k7QrX6hjA](https://user-images.githubusercontent.com/8519469/132960683-f3552ac3-b7b4-45fc-ba6a-21a1e6ed765a.jpg)

**Reply by abitofevrything** [reply]
Simply add the ~renderNote=... with ... replaced with a note with the html to be rendered (set that note's type to code/html)
